distribution/kpkginstall/runtest.sh

```diff
@@ -145,10 +145,17 @@ EOF
     exit 1
   fi
 
-  $YUM install -y kernel-firmware kernel-$KVER kernel-headers-$KVER >>${OUTPUTFILE} 2>&1
+  $YUM install -y kernel-$KVER kernel-headers-$KVER >>${OUTPUTFILE} 2>&1
   if [ $? -ne 0 ]; then
-    return 1
+    echo "Failed to install kernel!" | tee -a ${OUTPUTFILE}
+    exit 1
+  fi
+  $YUM install -y kernel-firmware >>${OUTPUTFILE} 2>&1
+  if [ $? -ne 0 ]; then
+    echo "No kernel-firmware available for install, skipping" | tee -a ${OUTPUTFILE}
```

Comment:
Maybe something goes wrong with the installation and this message will be confusing